### ISSUE TITLE
Upgrade prisma 2.12～

### ISSUE THREAD

**Issue by redblue9771** [prisma/migrate#655]
## Bug description

The console reports an error and cannot update the database.

## How to reproduce

run `npx prisma migrate save --experimental`
## Expected behavior

pass
## Prisma information

```
model Account {
  id        Int       @id @default(autoincrement())
  profile   User?
}

model User {
  id         Int      @id @default(autoincrement())
  account    Account  @relation(fields: [accountId], references: [id])
  accountId  Int      @unique
}
```

## Environment & setup

- Node.js version:  14
- Prisma version: 2.12

```
Error: The migration produced an invalid schema.
Datamodel error in schema that could not be rendered. Error parsing attribute "@relation": The relation field `account` on Model `User` is required. This is invalid as it is not possible to enforce this constraint at the database level. Please make it optional instead.
```

**Comment by redblue9771** [comment]
The problem was solved after deleting the old migration records

**Comment by redblue9771** [comment]
The problem was solved after deleting the old migration records

**Comment by rhlsthrm** [comment]
Where did you delete the migration records?

**Comment by redblue9771** [comment]
> Hey @redblue9771, I'm experiencing the issue as well, what exactly did you do to delete the migration records?

My operation is dangerous. This will affect the migration record. I deleted the migrations directory under the prisma directory. Then run `npx prisma migrate save --name init --experimental.` command to reinitialize.

Perhaps this issue should be reopened. What do you think?

**Comment by rhlsthrm** [comment]
Thanks @redblue9771, this is what I was thinking of doing as a last resort. I actually resolved this myself by going into the `steps.json` file for my migrations and finding the actual migration step that set the field to `Required` and changed it to `Optional`. However, this was not obvious at all and took me a few hours of debugging so I think it does make sense to reopen the issue.

**Comment by redblue9771** [comment]
> Thanks @redblue9771, this is what I was thinking of doing as a last resort. I actually resolved this myself by going into the `steps.json` file for my migrations and finding the actual migration step that set the field to `Required` and changed it to `Optional`. However, this was not obvious at all and took me a few hours of debugging so I think it does make sense to reopen the issue.

Hope it will be officially resolved soon~

**Comment by tomhoule** [comment]
Thanks for taking the time to report this issue!

We've recently released a Prisma Migrate Preview ([2.1.3.0 release notes](https://github.com/prisma/prisma/releases/tag/2.13.0)), which has quite a few changes compared to the previous experimental version. We believe this issue is no longer relevant in this new version, so we are closing this.

We would encourage you to try out Prisma Migrate Preview. If you come across this or any other issue in the preview version, please feel free to open a [new issue in prisma/prisma](https://github.com/prisma/prisma/issues/new/choose).
For general feedback on Prisma Migrate Preview, feel free to chime in [on this issue](https://github.com/prisma/prisma/issues/4531).
